client/me/get-apps/desktop-download-card.jsx

```diff
@@ -0,0 +1,191 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { identity } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+class DesktopDownloadCard extends Component {
+	static propTypes = {
+		translate: PropTypes.func,
+	};
+
+	static defaultProps = {
+		translate: identity,
+	};
+
+	constructor( props ) {
+		super( props );
+
+		this.windowsLink = 'https://apps.wordpress.com/d/windows?ref=getapps';
+		this.macLink = 'https://apps.wordpress.com/d/osx?ref=getapps';
+		this.linuxTarLink = 'https://apps.wordpress.com/d/linux?ref=getapps';
+		this.linuxDebLink = 'https://apps.wordpress.com/d/linux-deb?ref=getapps';
+	}
+
+	trackWindowsClick = () => recordTracksEvent( 'calypso_app_download_windows_click' );
+	trackMacClick = () => recordTracksEvent( 'calypso_app_download_mac_click' );
+	trackLinuxTarClick = () => recordTracksEvent( 'calypso_app_download_linux_tar_click' );
+	trackLinuxDebClick = () => recordTracksEvent( 'calypso_app_download_linux_deb_click' );
+
+	getDescription = ( platform ) => {
+		const { translate } = this.props;
+		switch ( platform ) {
+			case 'MacIntel':
+			case 'Linux i686':
+			case 'Linux i686 on x86_64':
+				return translate( 'A desktop app that gives WordPress a permanent home in your dock.' );
+			default:
+				return translate( 'A desktop app that gives WordPress a permanent home in your taskbar.' );
+		}
+	}
+
+	getRequirementsText = ( platform ) => {
```

Comment:
Also note that we only need to use `=>` for functions that will be passed to other components, like say an onClick handler.